nixos/modules/services/networking/ctrld.nix

```
@@ -0,0 +1,86 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ctrld;
+  settingsFormat = pkgs.formats.toml {};
+in {
+
+  options.services.ctrld = {
+    enable = mkEnableOption "ctrld service";
```

Maybe this could be a bit more descriptive? I think the grammar of the option sounds a bit weird as well, considering the content gets appended to "Whether to enable"

How about something like this?

```suggestion
    enable = mkEnableOption "ctrld, a highly configurable DNS forwarding proxy";
```

---

Done.